test/validation/org/apache/cassandra/modules/SimpleWriteModule.java

```diff
@@ -64,6 +64,18 @@ public void run()
                 session.execute(bound);
             }
 
+            try
+            {
+                JMXConnection myJMX = new JMXConnection("localhost", 7200, null, null);
+                Integer metric = myJMX.getObjectMetric("org.apache.cassandra.metrics:type=CQL,name=PreparedStatementsCount", "Value");
+                System.out.println(metric);
+                myJMX.close();
+            }
+            catch (IOException e)
+            {
+                throw new RuntimeException(e);
```

Comment:
I don't quite like having to catch this IOException here, but I think it is most technically correct for `connect()` to throw it. For now it's fine, I'll find a way to clean it up later so we don't need to do that.